core/camel-core-model/src/main/java/org/apache/camel/model/rest/RestDefinition.java

```diff
@@ -606,6 +619,10 @@ public RestDefinition security(String key, String scopes) {
         return this;
     }
 
+    public Collection<SecurityDefinition> securityRequirements() {
```

Comment:
What is this method for?

Reply:
This comes from https://github.com/apache/camel/blob/main/core/camel-core-model/src/main/java/org/apache/camel/model/rest/SecurityRequirementsDefinition.java#L76-L78